core/src/data/value/mod.rs

```diff
@@ -1591,6 +1628,142 @@ mod tests {
         test!(modulo I128(6),   F64(2.0) => F64(0.0));
         test!(modulo I128(6),   F32(2.0_f32) => F32(0.0_f32));
 
+        test!(bitwise_shift_left I8(1),     I64(2) => I8(4));
```

Comment:
at the time that we combine arithmetic binary operator unit tests into a single unit tests, we didn't know that the size of tests will grow like this big.

now rather than making this unit tests bigger, it would be better to make the unit test only for `bitwise_shift_left`.
Whether to use re-use this macro or not is up to you.